tests/admin_filters/tests.py

```diff
@@ -1828,7 +1828,7 @@ def test_list_filter_queryset_filtered_by_default(self):
         request.user = self.alfred
         changelist = modeladmin.get_changelist_instance(request)
         changelist.get_results(request)
-        self.assertEqual(changelist.full_result_count, 4)
+        self.assertEqual(changelist.full_result_count, 3)
```

Comment:
I think that explains why we can't make merge this change; if `list_filter` performs some dynamic filtering then `full_result_count` will have the wrong value.